<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShiftsDetailTab</class>
 <widget class="QWidget" name="ShiftsDetailTab">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="header_layout">
     <item>
      <widget class="QPushButton" name="btn_back_to_calendar">
       <property name="text">
        <string>&lt; Back to Calendar</string>
       </property>
       <property name="maximumWidth">
        <number>150</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer_left">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbl_shift_date_header">
       <property name="font">
        <font>
         <pointSize>14</pointSize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Shifts for Date</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer_right">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
       <widget class="QWidget" name="dummy_right">
        <property name="maximumWidth"><number>150</number></property>
       </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="table_shifts">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QHeaderView::section { border: 1px solid #6c6c6c; }</string>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Start</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>End</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Employee</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Hourly Rate</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Actions</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottom_layout">
     <item>
      <widget class="QPushButton" name="btn_refresh">
       <property name="text">
        <string>Refresh</string>
       </property>
       <property name="minimumHeight">
        <number>45</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_add_shift">
       <property name="text">
        <string>Add Shift</string>
       </property>
       <property name="minimumHeight">
        <number>45</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>